javascript/src/entrypoint.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+set -e
+
+ 
+# The follwing functions are taken from commonWrapper.sh
```

Comment:
We should move these to  an even common-er script.

Reply:
for example in a new directory 'common' instead of java/common

Reply:
Moved it like this: https://github.com/debricked/vulnerable-functionality/pull/5/commits/574cff768b4b7d8937ccc386d3058f4e6889e95e